AppInspector/rules/default/cloud_services/cloud_hosting.json

```diff
@@ -421,19 +367,14 @@
     "name": "Cloud Service: CI (Azure Pipelines)",
     "id": "AI002300",
     "description": "Cloud Service: CI (Azure Pipelines)",
-    "applies_to": [
-      "yaml"
-    ],
-    "tags": [
-      "Metadata.CloudServices.Code.CI.Microsoft.Azure"
-    ],
+    "applies_to":[ "yaml" ],
+    "tags":[ "Metadata.CloudServices.Code.CI.Microsoft.Azure" ],
```

Comment:
I think we should remove all of the `Metadata.` prefixes in the tags, thoughts?

Reply:
Some of the Metadata tags are magic values, so we can't do it in bulk.

https://github.com/microsoft/ApplicationInspector/blob/cecf0a2a3b5ad304a2763eab2da4e30df2b3b81c/AppInspector/MetaDataHelper.cs#L140

Reply:
Got it, makes sense.

Reply:
I can manually go through an remove the ones that aren't these magic values if you think that makes sense. Let me know.

Reply:
Done.